tests/e2e/suites/basic/basic_suite_test.go

```diff
@@ -0,0 +1,37 @@
+package basic
+
+import (
+	"testing"
+
+	// . "github.com/onsi/ginkgo/v2"
+	// . "github.com/onsi/gomega"
+
+	"github.com/giantswarm/apptest-framework/pkg/config"
+	"github.com/giantswarm/apptest-framework/pkg/suite"
+)
+
+const (
+	isUpgrade = false
+)
+
+func TestBasic(t *testing.T) {
+	suite.New(config.MustLoad("../../config.yaml")).
+		// The namespace to install the app into within the workload cluster
+		WithInstallNamespace("kube-system").
+		// If this is an upgrade test or not.
+		// If true, the suite will first install the latest released version of the app before upgrading to the test version
+		WithIsUpgrade(isUpgrade).
+		WithValuesFile("./values.yaml").
+		BeforeInstall(func() {
+			// Do any pre-install checks here (ensure the cluster has needed pre-reqs)
```

Comment:
I think you want to add things here :D

Reply:
It's a draft...

Reply:
But I should not be notified about drafts :(

Reply:
It's because I created it non draft... :-)